app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,20 @@
+class AdminController < ApplicationController
```

Comment:
Since there might be *lots* of admin paths, it's generally more conventional to do one of the following:
1. Create an admin folder under controllers, and then under the admin folder, make whatever controllers named after the resources they're dealing with - i.e. admin/shelters_controller, admin/applications_controller etc. WHen you set it up this way, your class declaration becomes `class Admin::SheltersController` to indicate that it lives in the admin folder
2. name the controllers with admin in the name: admin_shelters_controller, admin_applications_controller

In both of these cases, controllers are still based off the data they're manipulating (i.e. shelters or applications) but not just "admin".